components/hacbs/integration-service/base/manager_resources_patch.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: integration-service-controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        resources:
+          limits:
+            cpu: 500m
+            memory: 2Gi
+          requests:
+            cpu: 100m
+            memory: 20Mi
+        args:
+          - '--api-export-name=integration-service'
```

Comment:
I would wait for https://github.com/redhat-appstudio/integration-service/pull/55 to avoid duplication of args

Reply:
no this args is not needed

Reply:
Yes. Let's close it. Thank you @Michkov